IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -136,30 +136,28 @@ + (BOOL)handleChallenge:(NSURLAuthenticationChallenge *)challenge
     NSDictionary *queryItemsDict = [NSDictionary msidDictionaryFromQueryItems:queryItems];
     NSString *redirectURI = queryItemsDict[MSID_OAUTH2_REDIRECT_URI];
 
-    if (s_redirectScheme)
+    NSMutableDictionary *newQueryItems = [NSMutableDictionary new];
+    newQueryItems[MSID_BROKER_IS_PERFORMING_CBA] = @"true";
+    
+    for (NSURLQueryItem *item in queryItems)
```

Comment:
Don't block the PR on this, but would be great to add unit tests for this logic

Reply:
will add in separate pr